src/tasks/rebuild-web-api.js

```diff
@@ -295,10 +295,11 @@ module.exports = function rebuildWebApi(functionName, functionVersion, restApiId
 				}
 				if (additionalErrors()) {
 					additionalErrors().forEach(function (additionalError) {
+						var additionalErrorConfig = additionalError.toConfig();
```

Comment:
This would assume that we can only use the special config errors with this, and that's different from what we planned to do. The whole point of extracting errors into a separate package was so that people can just configure a few errors by hand if they want, and if someone wants to use generic utility handlers, they can.

until now, the config was just data, not functions. I'd try to keep that the same. I understand why you didn't like calling the .toConfig all the time, but perhaps we can find a different solution for that part in claudia-api-errors package.